packages/core/src/actions/smartAccount/types.ts

```diff
@@ -64,9 +70,11 @@ export type WaitForUserOperationTxParameters = {
 //#endregion WaitForUserOperationTxParameters
 
 //#region BuildUserOperationFromTransactionsResult
-export type BuildUserOperationFromTransactionsResult = {
-  uoStruct: UserOperationStruct;
+export type BuildUserOperationFromTransactionsResult<
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+> = {
+  uoStruct: UserOperationStruct<TEntryPointVersion>;
   batch: BatchUserOperationCallData;
-  overrides: UserOperationOverrides;
+  overrides: UserOperationOverrides<TEntryPointVersion>;
```

Comment:
what in overrides is entrypoint version dependent?

Reply:
paymaster data is different.